exporter/exporter.go

```diff
@@ -38,17 +38,19 @@ type Exporter struct {
 	onceRegisterUpdateAge      sync.Once
 	startup                    time.Time
 	promHandler                http.Handler
+	log                        logrus.Logger
```

Comment:
This should be a pointer. The semantics of such a type when copied as value are subtly dangerous. In this case there is a mutex hidden in that value which must not be copied after first use. Which it apparently is not, but it's better not summon bad ghosts.